install/src/Cabal.hs

```diff
@@ -84,18 +85,6 @@ cabalInstallHie versionNumber = do
              ++ minorVerExe
              ++ " to " ++ localBin
 
-installCabalWithStack :: Action ()
-installCabalWithStack = do
-  -- try to find existing `cabal` executable with appropriate version
-  mbc <- withoutStackCachedBinaries (liftIO (findExecutable "cabal"))
```

Comment:
I think we should keep this one for the reasons i gave in my comment.
In any case, if it is removed we should remove `withoutStackCachedBinaries` cause no one will be using it.